File: BRaVe-Mobile-Backend/DTOs/TokenRefreshRequest.cs
using System.Text.Json.Serialization;

namespace BRaVe_Mobile_Backend.DTOs
{
    public class TokenRefreshRequest
    {
        [JsonPropertyName("deviceId")]
        public string DeviceId { get; set; }

        [JsonPropertyName("enumerator")]
        public string Enumerator { get; set; }

        [JsonPropertyName("tokenRefresh")]
        public string TokenRefresh { get; set; }

        [JsonPropertyName("nonce")]
        public string Nonce { get; set; }

        [JsonPropertyName("dPoP")]
        public string DPoP { get; set; }
    }
}

File: BRaVeMobile/app/src/main/java/intl/iom/bravemobile/api/TokenRefreshRequest.java
package intl.iom.bravemobile.api;

import intl.iom.bravemobile.services.SecureStore;

public class TokenRefreshRequest {
    private String deviceId;
    private String enumerator;
    private String tokenRefresh;
    private String nonce;
    private String dPoP;


    // Getters and Setters
    public String getDeviceId() {
        return deviceId;
    }

    public void setDeviceId(String deviceId) {
        this.deviceId = deviceId;
    }

    public String getEnumerator() {
        return enumerator;
    }

    public void setEnumerator(String enumerator) {
        this.enumerator = enumerator;
    }

    public String getTokenRefresh() {
        return tokenRefresh;
    }

    public void setTokenRefresh(String tokenRefresh) {
        this.tokenRefresh = tokenRefresh;
    }

    public String getNonce() {
        return nonce;
    }

    public void setNonce(String nonce) {
        this.nonce = nonce;
    }

    public String getDPoP() {
        return dPoP;
    }

    public void setDPoP(String dPoP) {
        this.dPoP = dPoP;
    }
}